.travis.yml

```diff
@@ -1,14 +1,26 @@
-language: python
-
-python:
-  - "2.7"
+language: generic
 
 addons:
   postgresql: '9.4'
 
 before_script:
-    - yes | python setup.py install
-    - pip install -r dev-requirements.txt
-    - python bin/destroy_and_setup_psqlgraph.py
-    - pip freeze
-script: "py.test -v"
+  - psql -U postgres -c "create user test with superuser password 'test';"
+  - psql -U postgres -c "create database automated_test with owner test;"
+  - unset PYENV_ROOT
+  - rm -rf ~/.pyenv
+  - curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
```

Comment:
I agree with this recommendation in general.

For this specific case, I think it would be better to remove pyenv entirely and run separate Travis builds using the pre-baked Python 3.5/3.6 images. I'm not sure if the team is fully in agreement on this point, though.